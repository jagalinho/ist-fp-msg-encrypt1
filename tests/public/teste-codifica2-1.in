chave = (('A', 'B', 'C', 'D', 'E'), ('F', 'G', 'H', 'I', 'J'), ('L', 'M', 'N', 'O', 'P'), ('Q', 'R', 'S', 'T', 'U'), ('V', 'X', 'Z'))
cad = 'FUNDAMENTOS DA PROGRAMACAO'
cad_encriptada = codifica2(cad, chave)
cad_encriptada
