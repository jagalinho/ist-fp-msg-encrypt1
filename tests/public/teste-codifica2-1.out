'1034220300210422332332XX0300XX2431231131002100020023'
